src/PKSim.Infrastructure/ProjectConverter/v7_3/Converter721To730.cs

```diff
@@ -180,6 +191,12 @@ private void convertCompound(Compound compound)
             }
          }
 
+         foreach (var process in compound.AllProcesses())
```

Comment:
This will defacto update all process parameters dynamically. In case of parameter such as Volume liver exp etc.. they will be marked as changed after conversion. A small price to pay to get the right behavior